docs/source/guides/staking_guide.rst

```diff
@@ -59,6 +60,20 @@ All staking-related operations done by StakeHolder are performed through the ``n
 | ``--hw-wallet`` | Use a hardware wallet                      |
 +-----------------+--------------------------------------------+
 
+**ReStake Command Options**
+
++-------------------------+--------------------------------------------+
+| Option                  |  Description                               |
++=========================+============================================+
+|  ``--enable``           | Enable restaking (default)                 |
++-------------------------+--------------------------------------------+
+|  ``--disable``          | Disable restaking                          |
++-------------------------+--------------------------------------------+
+|  ``--lock ``            | Enable restaking lock                      |
++-------------------------+--------------------------------------------+
+|  ``--release-period``   | Expiration period of restaking lock        |
```

Comment:
In this block of green lines, I'd use "re-stake" (with hyphen), as you did in line 46.

Reply:
:heavy_check_mark: